requirements_dev_optional.txt

```
@@ -11,7 +11,7 @@ redis==3.3.8
 pymongo==3.9.0
 # optional test requirements
 tox==3.14.0
-coverage==4.5.4
+coverage==5.0.3
 coveralls==1.8.2
```

Maybe we can update this too?

```suggestion
coveralls==1.11.1
```

---

Yep, was just looking at this. Just waiting to see if this works or not, then going to try that. :)

---

Okay, looks like it worked with `debug` (but the service name is potentially wrong, let's see...).